.github/workflows/int-test-workflow.yml

```diff
@@ -4,6 +4,7 @@ on:
     branches:
       - develop
       - master
+      - cspl_1769
```

Comment:
I am guessing this needs to be removed after tests pass?

Reply:
Yes, will be removed.